docs/customising/make_a_new_theme.md

```diff
@@ -0,0 +1,258 @@
+
+<div class="attention-box warning">
+  You <strong>must</strong> rename the repo, because you'll run into problems
+  later on if <em>your</em> theme has the same name as the default one,
+  <code>alavetelitheme</code>.
+</div>
+
+## 3. Point your config at your new repo
+
+Now your own theme repo exists, point your Alaveteli install at it by editing
+the
+<code><a href="{{ page.baseurl }}/docs/customising/config/#theme_urls">THEME_URLS</a></code>
+setting in the config file.
+
+Set it to the URL of your uniquely named repo, for example:
+
+    THEME_URLS:
+         - 'git://github.com/yourgithub/abcexample-theme.git'
+
+## 4. Tell Alaveteli to get its theme
+
+Next, tell Alaveteli to pull the repo down from the URL in `THEMES_URL`, and
+install it as the theme your installation will use.
+
+Do this by issuing this command within your Vagrant virtual machine:
+
+    rake themes:install
+
+Alaveteli will connect to GitHub, pull down your theme repo and put it in the
+right place within your Alaveteli installation.
+
+When it's finished, you can see that your theme (that is, all the files from
+the repo) has been reproduced in `lib/themes/`.
+
+## 5. Change the primary colour
+
+You're ready to make a change and see that change appear in your browser.
+
+Alaveteli uses a set of basic <a href="{{ page.baseurl }}/docs/glossary/#sass"
+class="glossary__link">Sass</a> modules to define the layout for the site on
+different device sizes, and some basic styling.
+
+There's more than one way to do this, but the simplest way (which means you
+can most easily see the consequences of any changes you make) is to edit your
+theme's files inside Alaveteli installation, within `lib/themes/`. Within that
+directory, find your theme and edit
+`/assets/stylesheets/responsive/custom.scss`. Find the Sass variable `$primary`
+and change the colour value. For example:
+
+    $primary = #ff0000
+
+This sets the primary colour to red (`#ff0000` in that example can be any CSS
+colour value). Of course, this is just to show that you _can_ change it: really, you can change anything in this theme — that's the whole point.
+
+The development server automatically rebuilds any resources that have been
+changed; in this case it means it will build new versions of the CSS file from
+the Sass, containing the new colour you've picked.
+
+<div class="attention-box helpful-hint">
+  <p>
+    If you're new to git and GitHub, skip this note!
+  </p>
+  <p>
+    If you're already comfortable working with git, you might prefer to work in
+    the theme repo quite separately from the Alaveteli installation you're
+    deploying it into. That is, instead of editing the "deployed" theme files
+    within your development installation's <code>lib/themes/</code>, you can
+    work on a cloned copy of the theme anywhere (perhaps elsewhere on your
+    local machine, or even on GitHub), and then pull them into the development
```

Comment:
this might be a good place to suggest cloning the theme repo to an `alaveteli-themes` directory on the same level as the `alaveteli` repo directory as this is automatically discovered by Vagrant and the themes directories it contains (and any edits made to them) are available to the Alaveteli instance